### DISCUSSION TITLE
Creality has released the source code for the E3S1

### DISCUSSION THREAD

**Discussion by Caouette1988** [mriscoc/Ender3V2S1#73] [General]
https://github.com/CrealityOfficial/Ender-3S1

**Comment by mriscoc** [comment]
Thanks, yes, we are porting some configuration values and reviewing the source code. Did you saw the line 1488 of the Configuration.h file? They forgot to delete something there 😊

**Reply by Xrayzor1337** [reply]
I saw your contribution...nice job!
